test/mirage/query/run.t

```diff
@@ -4,73 +4,128 @@ Query name
   $ ./config.exe query name
   noop
 
-Query opam
-  $ ./config.exe query opam
+Query global opam
+  $ ./config.exe query global.opam
   opam-version: "2.0"
   name: "noop"
   maintainer: "dummy"
   authors: "dummy"
   homepage: "dummy"
   bug-reports: "dummy"
-  dev-repo: "git+https://example.com/nonexistent"
-  synopsis: "This is a dummy"
+  dev-repo: "git://dummy"
+  synopsis: "Unikernel noop - main opam file"
+  description: """
+  It assumes that local dependencies are already
+  fetched.
+  """
   
-  build: ["mirage" "build" "--config-file" "config.ml"]
+  build: [
+    [ "mirage" "configure"  ]
+    [ "mirage" "build" ]
+  ]
+  
+  install: [
+    [ "cp" "dist/noop" "%{bin}%/noop" ]
+  ]
   
   depends: [
-    "lwt"
-    "mirage" {build &  >= "3.10.0" & < "3.11.0"}
-    "mirage-bootvar-unix" { >= "0.1.0" & < "0.2.0"}
-    "mirage-clock-unix" { >= "3.0.0" & < "4.0.0"}
-    "mirage-logs" { >= "1.2.0" & < "2.0.0"}
-    "mirage-runtime" { >= "3.10.0" & < "3.11.0"}
-    "mirage-types" { >= "3.10.0" & < "3.11.0"}
-    "mirage-unix" { >= "4.0.0" & < "5.0.0"}
-    "ocaml" {build &  >= "4.08.0"}
-    "ocamlbuild" {build}
-    "ocamlfind" {build}
+    
   ]
   
 
+Query local opam
```

Comment:
So by reading that example, I got the local/global distinction wrong. @TheLortex can you try to clarify this a bit?

What's the difference between the local/global/install/locked opam files? :-)

Reply:
There are three opam files in a mirage project:
- `<name>.opam`: unikernel depencies that have to be locally fetched so that they exist in the dune workspace. Notably all runtime dependencies should be in that opam file, but it can also contain build tools as long as they build using dune. 
- `<name>.opam.locked`: unikernel dependencies lockfile obtained using `opam-monorepo`. 
- `<name>-install.opam`: the rest of the build tools, the main example being `ocaml-freestanding`. They are installed in the switch using opam. 

I agree that the names are a bit confusing but I don't know any other option. The rationale is that there are dependencies that need to be installed _globally_ in your opam switch and that there are runtime dependencies that need to be _locally_ fetched alongside the unikernel files as a monorepo in a dune workspace.

Reply:
New proposal: 
* _global_ becomes _switch_
* _local_ becomes _monorepo_
Therefore the names are more in sync with the traditional ocaml concepts (opam switch and opam-monorepo)

What do you think ?